certified-connectors/Sociabble/apiDefinition.swagger.json

```diff
@@ -1551,6 +2665,19 @@
             "title": "Media url"
           }
         }
+      },
+      "CtaSuggestContent":{
```

Comment:
@xavtesdev Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
@sriyen-msft @Amjed-Ayoub 
I resolved the mistake and fill missing informations